src/lycophron/cli.py

```diff
@@ -92,6 +240,113 @@ def configure():
     pass
 
 
+@lycophron.command()
+@click.option("--filename", prompt="Filename", default="data.xlsx")
+@click.option("--journal", default=False, is_flag=True)
+@click.option("--meeting", default=False, is_flag=True)
+@click.option("--imprint", default=False, is_flag=True)
+@click.option("--thesis", default=False, is_flag=True)
+@click.option("--dwc", default=False, is_flag=True)
+@click.option("--gbif-dwc", default=False, is_flag=True)
+@click.option("--ac", default=False, is_flag=True)
+@click.option("--dc", default=False, is_flag=True)
+@click.option("--openbiodiv", default=False, is_flag=True)
+@click.option("--obo", default=False, is_flag=True)
+@click.option("--all", default=False, is_flag=True)
+def new(
+    filename,
+    journal,
+    meeting,
+    imprint,
+    thesis,
+    dwc,
+    gbif_dwc,
+    ac,
+    dc,
+    openbiodiv,
+    obo,
+    all,
+):
+    """Creates a new CSV template."""
+    # Consolidate all fields
+    all_fields = {
+        key: _generate_fields(field_prefixes[key], fields)
+        for key, fields in field_definitions.items()
+    }
+
+    # Add base RDM fields to headers
+    headers = rdm_fields.copy()
+
+    # Add conditional fields
+    options = {
+        "journal": journal,
+        "meeting": meeting,
+        "imprint": imprint,
+        "thesis": thesis,
+        "dwc": dwc,
+        "gbif-dwc": gbif_dwc,
+        "ac": ac,
+        "dc": dc,
+        "openbiodiv": openbiodiv,
+        "obo": obo,
+    }
+
+    # Add conditional fields if enabled to headers
+    for option, is_enabled in options.items():
+        if all or is_enabled:
+            headers.extend(all_fields.get(option, []))
+
+    # Create workbook and add headers
+    workbook = openpyxl.Workbook()
+    sheet = workbook.active
+
+    for col, header in enumerate(headers, start=1):
+        sheet.cell(row=1, column=col, value=header)
+
+    workbook.save(filename)
+
+
+@lycophron.command()
+@click.option("--file", prompt="CSV File", type=click.Path(exists=True))
+def validate(file):
+    """Validates the config and headers of a CSV file."""
+    app = LycophronApp()
+    # Validates config
+    try:
+        app.validate_project()
+        click.echo(click.style("Configuration validation passed.", fg="green"))
+    except ValueError as e:
+        click.echo(click.style(e, fg="red"))
+        return
+
+    # Validates headers
+    df = pd.read_excel(file, header=None, nrows=1)
```

Comment:
minor: can we skip adding pandas? Since we're working with CSV,  the default `CSV` library can also retrieve the headers IIRC.

Reply:
Right, since these are simple operations I think the csv lib is enough, thanks!